tangos/scripts/manager.py

```diff
@@ -295,13 +295,46 @@ def add_tracker(halo, size=None):
         print("done")
     core.get_default_session().commit()
 
-
-def grep_run(opts):
-    run = core.get_default_session().query(Creator).filter(
+def grep_run_id(opts):
+    matching_runs = core.get_default_session().query(Creator).filter(
         Creator.command_line.like("%" + opts.query + "%")).all()
-    for r in run:
+    for r in matching_runs:
         print(r.id, end=' ')
-
+    print('\n')
+        
+def grep_run_info(opts):
+    matching_runs = core.get_default_session().query(Creator).filter(
+        Creator.command_line.like("%" + opts.query + "%")).all()
+    for r in matching_runs:
+        r.print_info()
+    return matching_runs
+        
+def grep_remove_runs(opts):
+    print('Will remove the following run(s): ')
+    matching_runs = grep_run_info(opts)
+    
+    # Always ask for confirmation, as this action could be very destructive
+    print("You want to delete everything created by the above run(s)?")
+    print(""">>> type "yes" to continue""")
+    
+    if input(":").lower() == "yes":
+        
+        for r, run in enumerate(matching_runs):
+            print('Removing run ',r+1,' of ',len(matching_runs))
+            
+            run.halolinks.delete()
```

Comment:
This seems to be a copy/paste from `rem_run` -- I think they should both call a function so as not to duplicate code

Reply:
It very much is a copy-paste, I will wrap it up in a function.